luminoso_api/json_stream.py

```diff
@@ -254,6 +260,13 @@ def _read_csv(reader, header, encode_fn):
         yield row_dict
 
 
+def convert_date(date_string, date_format):
```

Comment:
This function could (a) use a docstring; (b) be moved closer to the function that actually uses it; (c) grow a leading underscore, if it is only an internal function.